packages/main-api/key.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export cdir=$(pwd)
+
+# if ../../.build/evm-keys, run `make .build/evm-keys` 
+if [ ! -f ../../.build/evm-keys ]; then
+    cd ../../
+    make .build/evm-keys > /dev/null 2>&1
+    cd $cdir
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)